plugins/org.jboss.reddeer.core/src/org/jboss/reddeer/core/condition/ShellMatchingMatcherIsAvailable.java

```diff
@@ -39,13 +38,20 @@ public ShellMatchingMatcherIsAvailable(Matcher<?>... matchers) {
 		AndMatcher am  = new AndMatcher(matchers);
 		this.matcher = am;
 	}
+	
+	/**
+	 * Creates new ShellMatchingMatcherIsAvailable wait condition with specified shell text.
+	 * @param text shell text
+	 */
+	public ShellMatchingMatcherIsAvailable(String text){
```

Comment:
Another option is to move the merged ShellIsAvailable to
```
org.jboss.reddeer.swt.condition
```
Here we could use String, Matcher and RedDeer Shell.

IMHO this is the best solution since the package
```
org.jboss.reddeer.core.condition
```
should not use any condition with SWT dependencies (e.g. SWT Shell).